### DISCUSSION TITLE
Intermittent Failure of Daily Backup

### DISCUSSION THREAD

**Discussion by zkygr** [nextcloud/all-in-one#4216] [Questions] [answered]
### Description 

Hello,

I've encountered a recurring issue with the daily backup feature. 
The problem persists with a consistent pattern: initially, the backup process runs smoothly without any hiccups. 
However, after approximately 3-7 days, the backup fails to execute. 

Interestingly, logging into the interface of the master container seems to trigger the backup process to resume. 
Subsequently, the backup operates smoothly for a few more days before encountering the same issue again.

### Host OS 

- Debian 12

#### Nextcloud AIO version 

- Nextcloud AIO v7.11.2->

**Comment by szaimen** [comment, marked as answer]
Hi, I cannot reproduce the problem unfortunately.

However I think someone reported a comparable issue in the past which was caused by a mountpoint that was not correctly mounted and thus stopped the backup container from running.

**Comment by andreas-1107** [comment]
Hello, I thought I would report something similar in case it helps others. The intermittent failure happens for me regularly too. The AIO logs say nothing useful, eg most recently: 

**# docker logs nextcloud-aio-database**

```
++ rm -f /mnt/data/database-dump.sql.temp
++ touch /mnt/data/export.failed
++ pg_dump --username nextcloud nextcloud_database
++ rm -f /mnt/data/database-dump.sql
++ mv /mnt/data/database-dump.sql.temp /mnt/data/database-dump.sql
++ pg_ctl stop -m fast
2026-01-02 02:01:18.097 CET [14] LOG:  received fast shutdown request
2026-01-02 02:01:18.100 CET [14] LOG:  aborting any active transactions
2026-01-02 02:01:18.111 CET [14] LOG:  background worker "logical replication launcher" (PID 27) exited with exit code 1
2026-01-02 02:01:18.111 CET [22] LOG:  shutting down
2026-01-02 02:01:18.217 CET [14] LOG:  database system is shut down
waiting for server to shut down.... done
server stopped
++ rm /mnt/data/export.failed
++ echo 'Database dump successful!'
Database dump successful!
++ set +x
```

or the master container

**# docker logs nextcloud-aio-mastercontainer**

```
Waiting for backup container to stop
Waiting for backup container to stop
Waiting for backup container to stop
...
Waiting for backup container to stop
Waiting for backup container to stop
Waiting for backup container to stop
...
Waiting for backup container to stop
Waiting for backup container to stop
Waiting for backup container to stop
```


It's definitely a hardware issue. I use ZFS as filesystem and after each time, the ZFS disk being backed up to (disk-bak) is suspended. I thought it was due to a faulty SSD, but I recently switched it out to a new one. My next guess is that it's something bad with the USB hub I'm using.

**# zpool list**

```
NAME      SIZE  ALLOC   FREE  CKPOINT  EXPANDSZ   FRAG    CAP  DEDUP    HEALTH  ALTROOT
disk-data  1.81T   730G  1.10T        -         -     1%    39%  1.00x    ONLINE  -
disk-bak  1.81T   818G  1.01T        -         -     7%    44%  1.00x  SUSPENDED  -

```
